ladybug/debugger/src/main/java/org/frankframework/ladybug/DeploymentSpecificsBeanPostProcessor.java

```diff
@@ -41,6 +41,11 @@ public Object postProcessBeforeInitialization(Object bean, String beanName) thro
 				long maxStorageSizeLong = OptionConverter.toFileSize(maxStorageSize, databaseStorage.getMaxStorageSize());
 				databaseStorage.setMaxStorageSize(maxStorageSizeLong);
 			}
+			String maxStorageDays = appConstants.getProperty("ibistesttool.maxStorageDays");
+			if (maxStorageDays != null) {
+				long maxStorageDaysLong = OptionConverter.toFileSize(maxStorageDays, -1);
+				databaseStorage.setMaxStorageDays(maxStorageDaysLong);
+			}
```

Comment:
Men moet tzt wel echt overgaan op Spring `@Value`s want dit is niet zo mooiii...